index.js

```diff
@@ -1,24 +1,37 @@
 const Logr = require('logr');
 const logrFlat = require('logr-flat');
+const logfmt = require('logr-logfmt');
 const wreck = require('wreck');
 const humanDate = require('human-date');
 const runshell = require('runshell');
 const confi = require('confi');
 const CronJob = require('cron').CronJob;
-const log = Logr.createLogger({
-  type: 'flat',
-  reporters: {
-    flat: {
-      reporter: logrFlat,
-      options: {
-        timestamp: false,
-        appColor: true,
-        theme: {
-          keys: 'cyan'
-        }
+
+const reporters = {
+  flat: {
+    reporter: logrFlat,
+    options: {
+      timestamp: false,
+      appColor: true,
+      theme: {
+        keys: 'cyan'
       }
     }
   }
+};
+
+if (process.env.LOGFMT) {
+  reporters.logfmt = {
```

Comment:
lets just use logfmt all the time (remove all flat stuff). color is dictated by the env var, so don't overrride